IPs/RSKIP207.md

```diff
@@ -15,36 +15,40 @@
 
 The time-locked emergency multisignature introduced in RSKIP201 requires that Powpeg UTXOs are periodically spent in order to prevent the time-lock expiration. 
 
-This RSKIP proposes a mechanism for the Bridge to command this time-lock refresh in a way that maintains high-availability of funds for peg-out, and low risk of time-lock expiration.
+This RSKIP proposes a mechanism for the Bridge to command this time-lock refresh efficiently.
 
 
 
 ## Motivation
 
-The motivation for adding an time-locked emergency multisignature is presented in RSKIP201.
+The motivation for adding an time-locked emergency multisignature is presented in [RSKIP201](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP201.md).
 
 ## Specification
 
-The bridge contract maintains a FIFO queue of UTXOs called **oldUTXOs** as a double-linked list, where Powpeg UTXOs are ordered by block height of inclusion in the Bitcoin blockchain. The Bridge maintains an **oldest** and **newest** pointers to the first and last elements. Every time updateCollections() is called a new procedure **checktimeLockExpiration()** is called before any other processing. In this procedure checks are performed in sequence:
+Let B be the Bitcoin blockchain height as seen by the bridge. We define a UTXO that needs **refresh** to be one that is included in the Bitcoin blockchain at block height C where (B-C > P) where P is 26000 (a rounded average number of Bitcoin blocks produced over a 6 months period). 
 
-1. Check if composition voting phase should be cancelled.
+The bridge contract maintains a FIFO queue of UTXOs called **oldUTXOs** as a double-linked list, where Powpeg UTXOs are ordered by block height of inclusion in the Bitcoin blockchain. The Bridge maintains an **oldest** and **newest** pointers to the first and last elements in the queue. Every time updateCollections() is called a new procedure **checktimeLockExpiration()** is called before any other processing takes place. This method performs the following actions:
 
-2. Check if its not undergoing a membership change protocol and if a timestamp D has been reached.
+1. Let H be a storage variable containing an check point expressed as an RSK block height. Let D be a storage variable containing a checkpoint expressed as a timestamp. Let B be the block height of the tip of the Bitcoin blockchain as seen by the bridge. If the current RSK block height is higher than H or the last RSK block timestamp is higher than D, then continue, else exit the checktimeLockExpiration() method.
```

Comment:
* Will be good to explain what an "expiration checkpoint" is
* In this line "check point -> checkpoint"